Sources/Sentry/Public/SentryOptions.h

```diff
@@ -130,6 +130,14 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, readonly, strong) SentrySdkInfo *sdkInfo;
 
+/**
+ * The maximum size for each attachment in MiB. Default is 5 MiB.
```

Comment:
What about we make the default much larger like 20? Should be possible to attach 20mb of compressed data

Reply:
I'm fine with 20.